docs: add content on ksqlDB CLI usage (#7338)
confluentinc/ksql: docs/operate-and-deploy/installation/cli-config.md

--------------------------------

You can set the properties by using the ksqlDB CLI startup script argument
`/bin/ksql --config-file <path/to/file> -- <ksql-server-URL>` or by using the SET
statement from within the ksqlDB CLI session. For more information, see
[Start the ksqlDB CLI](installing.md#start-the-ksqldb-cli).

Here are some common ksqlDB CLI properties that you can customize:

-   [ksql.streams.auto.offset.reset](../../reference/server-configuration.md#ksqlstreamsautooffsetreset)
-   [ksql.streams.cache.max.bytes.buffering](../../reference/server-configuration.md#ksqlstreamscachemaxbytesbuffering)
-   [ksql.streams.num.stream.threads](../../reference/server-configuration.md#ksqlstreamsnumstreamthreads)

Connect to an Auth-enabled ksqlDB server
----------------------------------------

To connect to a ksqlDB server that requires authentication, supply your username
and password when starting the ksqlDB CLI:

```bash
/bin/ksql -u <username> -p <password> <ksql-server-URL>
```

> If you don't wish to supply your password as part of the command, you can

Reviewer: ```suggestion
If you don't want to supply your password as part of the command,
```

> instead supply just the username, and you will be prompted for your password

Reviewer: ```suggestion
supply just the username, and you'll be prompted for your password
```
